packages/button/src/Button.js

```diff
@@ -171,18 +147,17 @@ const buttonStyles = (props, baseTheme) => {
     cursor: props.disabled ? 'default' : 'pointer',
     fontWeight: theme.Button_fontWeight,
     height: theme[`Button_size_${props.size}`],
-    padding: props.circular
-      ? `${paddingTopBottom} 0`
-      : `${paddingTopBottom} ${paddingLeftRight}`,
+    // padding: `${parseFloat(theme[`Button_padding_${props.size}`]) -
+    //   parseFloat(pxToEm(theme.Button_borderWidth))}em`,
+
+    padding: ((
```

Comment:
Agreed. I went back to the previous code and added an explanatory comment to be extra clear.